dev/docker/gravitino/gravitino-dependency.sh

```diff
@@ -67,6 +67,9 @@ echo "Start to download the jar package"
 mysql_driver="mysql-connector-java-8.0.27.jar"
 wget "https://repo1.maven.org/maven2/mysql/mysql-connector-java/8.0.27/$mysql_driver" -O "${gravitino_staging_dir}/${mysql_driver}"
 cp "${gravitino_staging_dir}/${mysql_driver}" "${gravitino_package_dir}/catalogs/jdbc-mysql/libs/"
+cp "${gravitino_staging_dir}/${mysql_driver}" "${gravitino_package_dir}/catalogs/jdbc-doris/libs/"
+cp "${gravitino_staging_dir}/${mysql_driver}" "${gravitino_package_dir}/catalogs/jdbc-starrocks/libs/"
+cp "${gravitino_staging_dir}/${mysql_driver}" "${gravitino_package_dir}/catalogs/jdbc-oceanbase/libs/"
```

Comment:
should also update `docker-image-details.md`?

Reply:
Not necessarily。 As usual, we'll be posting a new mirror with each release and the message will be similar as the follow:

https://github.com/apache/gravitino/blob/eba4808c2eac97dafdc0fd8bb64cc6df751f117f/docs/docker-image-details.md?plain=1#L23-L42